source/geometries/Next100FieldCage.cc

```diff
@@ -603,11 +769,15 @@ void Next100FieldCage::BuildELRegion()
   if (visibility_) {
     G4VisAttributes light_blue = nexus::LightBlue();
     el_gap_logic->SetVisAttributes(light_blue);
-    diel_grid_logic->SetVisAttributes(G4VisAttributes::GetInvisible());
+    el_grid_logic->SetVisAttributes(G4VisAttributes::GetInvisible());
```

Comment:
If the EL grid is always invisible, you can move it outside the if/else clause, together with the attributes of gate and anode.